hcx/resource_vmc.go

```diff
@@ -85,18 +87,18 @@ func resourceVmcCreate(ctx context.Context, d *schema.ResourceData, m interface{
 		return diag.FromErr(err)
 	}
 
-	// Check if already activated
+	// Check if already activated.
 	if sddc.DeploymentStatus == "ACTIVE" {
```

Comment:
Looks like I might have missed one here.

Reply:
```suggestion
	if sddc.DeploymentStatus == constants.VmcActivationActiveStatus {
```

Reply:
Resolved in https://github.com/vmware/terraform-provider-hcx/commit/fde3d12345b41ae465b10675091096cd6e40f28c.